contracts/modules/royalty/policies/AncestorsVaultLAP.sol

```diff
@@ -69,8 +66,8 @@ contract AncestorsVaultLAP is IAncestorsVaultLAP, ERC1155Holder, ReentrancyGuard
     function _transferRnftsAndAccruedTokens(
         address claimerIpId,
         address splitClone,
-        address[] calldata ancestors,
-        uint32[] calldata ancestorsRoyalties,
+        address[] memory ancestors,
+        uint32[] memory ancestorsRoyalties,
```

Comment:
What was the cause for change from `calldata` to `memory`?

Reply:
Before data came from outside the function as calldata
Now it does not. It is read and saved in memory.

Reply:
Oh I see, missed the part where you loaded data from storage (replacing the outputs). Thanks